Description: GRUB tests fail to build on our hardenized build nodes. We disable them.

--- grub2-2.12.orig/Makefile.am
+++ grub2-2.12/Makefile.am
@@ -439,7 +439,7 @@ endif
 
 endif
 
-EXTRA_DIST += grub-core/tests/boot/kbsd.init-i386.S grub-core/tests/boot/kbsd.init-x86_64.S grub-core/tests/boot/kbsd.spec.txt grub-core/tests/boot/kernel-8086.S grub-core/tests/boot/kernel-i386.S grub-core/tests/boot/kfreebsd-aout.cfg grub-core/tests/boot/kfreebsd.cfg grub-core/tests/boot/kfreebsd.init-i386.S grub-core/tests/boot/kfreebsd.init-x86_64.S grub-core/tests/boot/knetbsd.cfg grub-core/tests/boot/kopenbsd.cfg grub-core/tests/boot/kopenbsdlabel.txt grub-core/tests/boot/linux16.cfg grub-core/tests/boot/linux.cfg grub-core/tests/boot/linux.init-i386.S grub-core/tests/boot/linux.init-mips.S grub-core/tests/boot/linux.init-ppc.S grub-core/tests/boot/linux.init-x86_64.S grub-core/tests/boot/linux-ppc.cfg grub-core/tests/boot/multiboot2.cfg grub-core/tests/boot/multiboot.cfg grub-core/tests/boot/ntldr.cfg grub-core/tests/boot/pc-chainloader.cfg grub-core/tests/boot/qemu-shutdown-x86.S
+#EXTRA_DIST += grub-core/tests/boot/kbsd.init-i386.S grub-core/tests/boot/kbsd.init-x86_64.S grub-core/tests/boot/kbsd.spec.txt grub-core/tests/boot/kernel-8086.S grub-core/tests/boot/kernel-i386.S grub-core/tests/boot/kfreebsd-aout.cfg grub-core/tests/boot/kfreebsd.cfg grub-core/tests/boot/kfreebsd.init-i386.S grub-core/tests/boot/kfreebsd.init-x86_64.S grub-core/tests/boot/knetbsd.cfg grub-core/tests/boot/kopenbsd.cfg grub-core/tests/boot/kopenbsdlabel.txt grub-core/tests/boot/linux16.cfg grub-core/tests/boot/linux.cfg grub-core/tests/boot/linux.init-i386.S grub-core/tests/boot/linux.init-mips.S grub-core/tests/boot/linux.init-ppc.S grub-core/tests/boot/linux.init-x86_64.S grub-core/tests/boot/linux-ppc.cfg grub-core/tests/boot/multiboot2.cfg grub-core/tests/boot/multiboot.cfg grub-core/tests/boot/ntldr.cfg grub-core/tests/boot/pc-chainloader.cfg grub-core/tests/boot/qemu-shutdown-x86.S
 
 windowsdir=$(top_builddir)/$(PACKAGE)-$(VERSION)-for-windows
 windowsdir: $(PROGRAMS) $(starfield_DATA) $(platform_DATA)
--- grub2-2.12.orig/Makefile.util.def
+++ grub2-2.12/Makefile.util.def
@@ -775,595 +775,6 @@ script = {
   mansection = 1;
 };
 
-script = {
-  name = grub-shell;
-  common = tests/util/grub-shell.in;
-  installdir = noinst;
-};
-
-script = {
-  name = grub-shell-tester;
-  common = tests/util/grub-shell-tester.in;
-  installdir = noinst;
-};
-
-script = {
-  name = grub-shell-luks-tester;
-  common = tests/util/grub-shell-luks-tester.in;
-  installdir = noinst;
-};
-
-script = {
-  name = grub-fs-tester;
-  common = tests/util/grub-fs-tester.in;
-  installdir = noinst;
-  dependencies = 'garbage-gen$(BUILD_EXEEXT)';
-};
-
-script = {
-  testcase = native;
-  name = ext234_test;
-  common = tests/ext234_test.in;
-};
-
-script = {
-  testcase = native;
-  name = squashfs_test;
-  common = tests/squashfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = iso9660_test;
-  common = tests/iso9660_test.in;
-};
-
-script = {
-  testcase = native;
-  name = hfsplus_test;
-  common = tests/hfsplus_test.in;
-};
-
-script = {
-  testcase = native;
-  name = ntfs_test;
-  common = tests/ntfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = reiserfs_test;
-  common = tests/reiserfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = fat_test;
-  common = tests/fat_test.in;
-};
-
-script = {
-  testcase = native;
-  name = minixfs_test;
-  common = tests/minixfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = xfs_test;
-  common = tests/xfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = f2fs_test;
-  common = tests/f2fs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = nilfs2_test;
-  common = tests/nilfs2_test.in;
-};
-
-script = {
-  testcase = native;
-  name = romfs_test;
-  common = tests/romfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = exfat_test;
-  common = tests/exfat_test.in;
-};
-
-script = {
-  testcase = native;
-  name = tar_test;
-  common = tests/tar_test.in;
-};
-
-script = {
-  testcase = native;
-  name = udf_test;
-  common = tests/udf_test.in;
-};
-
-script = {
-  testcase = native;
-  name = hfs_test;
-  common = tests/hfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = jfs_test;
-  common = tests/jfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = btrfs_test;
-  common = tests/btrfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = zfs_test;
-  common = tests/zfs_test.in;
-};
-
-script = {
-  testcase = native;
-  name = cpio_test;
-  common = tests/cpio_test.in;
-};
-
-script = {
-  testcase = native;
-  name = example_scripted_test;
-  common = tests/example_scripted_test.in;
-};
-
-script = {
-  testcase = native;
-  name = gettext_strings_test;
-  common = tests/gettext_strings_test.in;
-  extra_dist = po/exclude.pot;
-};
-
-script = {
-  testcase = nonnative;
-  name = pata_test;
-  common = tests/pata_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = ahci_test;
-  common = tests/ahci_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = uhci_test;
-  common = tests/uhci_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = ohci_test;
-  common = tests/ohci_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = ehci_test;
-  common = tests/ehci_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = example_grub_script_test;
-  common = tests/example_grub_script_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_eval;
-  common = tests/grub_script_eval.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_test;
-  common = tests/grub_script_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_echo1;
-  common = tests/grub_script_echo1.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_leading_whitespace;
-  common = tests/grub_script_leading_whitespace.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_echo_keywords;
-  common = tests/grub_script_echo_keywords.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_vars1;
-  common = tests/grub_script_vars1.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_for1;
-  common = tests/grub_script_for1.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_while1;
-  common = tests/grub_script_while1.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_if;
-  common = tests/grub_script_if.in;
-};
-
-script = {
-  testcase = native;
-  name = grub_script_blanklines;
-  common = tests/grub_script_blanklines.in;
-};
-
-script = {
-  testcase = native;
-  name = grub_script_final_semicolon;
-  common = tests/grub_script_final_semicolon.in;
-};
-
-script = {
-  testcase = native;
-  name = grub_script_dollar;
-  common = tests/grub_script_dollar.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_comments;
-  common = tests/grub_script_comments.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_functions;
-  common = tests/grub_script_functions.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_break;
-  common = tests/grub_script_break.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_continue;
-  common = tests/grub_script_continue.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_shift;
-  common = tests/grub_script_shift.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_blockarg;
-  common = tests/grub_script_blockarg.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_setparams;
-  common = tests/grub_script_setparams.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_return;
-  common = tests/grub_script_return.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_cmd_cryptomount;
-  common = tests/grub_cmd_cryptomount.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_cmd_regexp;
-  common = tests/grub_cmd_regexp.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_cmd_date;
-  common = tests/grub_cmd_date.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_cmd_set_date;
-  common = tests/grub_cmd_set_date.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_cmd_sleep;
-  common = tests/grub_cmd_sleep.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_expansion;
-  common = tests/grub_script_expansion.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_not;
-  common = tests/grub_script_not.in;
-};
-
-script = {
-  testcase = native;
-  name = grub_script_no_commands;
-  common = tests/grub_script_no_commands.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = partmap_test;
-  common = tests/partmap_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = hddboot_test;
-  common = tests/hddboot_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = fddboot_test;
-  common = tests/fddboot_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = cdboot_test;
-  common = tests/cdboot_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = netboot_test;
-  common = tests/netboot_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = serial_test;
-  common = tests/serial_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = pseries_test;
-  common = tests/pseries_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = core_compress_test;
-  common = tests/core_compress_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = xzcompress_test;
-  common = tests/xzcompress_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = gzcompress_test;
-  common = tests/gzcompress_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = lzocompress_test;
-  common = tests/lzocompress_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_cmd_echo;
-  common = tests/grub_cmd_echo.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = help_test;
-  common = tests/help_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_gettext;
-  common = tests/grub_script_gettext.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_escape_comma;
-  common = tests/grub_script_escape_comma.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_script_strcmp;
-  common = tests/grub_script_strcmp.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = test_sha512sum;
-  common = tests/test_sha512sum.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = test_unset;
-  common = tests/test_unset.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_func_test;
-  common = tests/grub_func_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_cmd_tr;
-  common = tests/grub_cmd_tr.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = file_filter_test;
-  common = tests/file_filter_test.in;
-};
-
-script = {
-  testcase = nonnative;
-  name = grub_cmd_test;
-  common = tests/grub_cmd_test.in;
-};
-
-script = {
-  testcase = native;
-  name = syslinux_test;
-  common = tests/syslinux_test.in;
-};
-
-script = {
-  testcase = native;
-  name = luks1_test;
-  common = tests/luks1_test.in;
-};
-
-script = {
-  testcase = native;
-  name = luks2_test;
-  common = tests/luks2_test.in;
-};
-
-program = {
-  testcase = native;
-  name = example_unit_test;
-  common = tests/example_unit_test.c;
-  common = tests/lib/unit_test.c;
-  common = grub-core/kern/list.c;
-  common = grub-core/kern/misc.c;
-  common = grub-core/tests/lib/test.c;
-  ldadd = libgrubmods.a;
-  ldadd = libgrubgcry.a;
-  ldadd = libgrubkern.a;
-  ldadd = grub-core/lib/gnulib/libgnu.a;
-  ldadd = '$(LIBDEVMAPPER) $(LIBZFS) $(LIBNVPAIR) $(LIBGEOM)';
-};
-
-program = {
-  testcase = native;
-  name = printf_test;
-  common = tests/printf_unit_test.c;
-  common = tests/lib/unit_test.c;
-  common = grub-core/kern/list.c;
-  common = grub-core/kern/misc.c;
-  common = grub-core/tests/lib/test.c;
-  ldadd = libgrubmods.a;
-  ldadd = libgrubgcry.a;
-  ldadd = libgrubkern.a;
-  ldadd = grub-core/lib/gnulib/libgnu.a;
-  ldadd = '$(LIBDEVMAPPER) $(LIBZFS) $(LIBNVPAIR) $(LIBGEOM)';
-};
-
-program = {
-  testcase = native;
-  name = date_test;
-  common = tests/date_unit_test.c;
-  common = tests/lib/unit_test.c;
-  common = grub-core/kern/list.c;
-  common = grub-core/kern/misc.c;
-  common = grub-core/tests/lib/test.c;
-  ldadd = libgrubmods.a;
-  ldadd = libgrubgcry.a;
-  ldadd = libgrubkern.a;
-  ldadd = grub-core/lib/gnulib/libgnu.a;
-  ldadd = '$(LIBDEVMAPPER) $(LIBZFS) $(LIBNVPAIR) $(LIBGEOM)';
-};
-
-program = {
-  testcase = native;
-  name = priority_queue_unit_test;
-  common = tests/priority_queue_unit_test.cc;
-  common = tests/lib/unit_test.c;
-  common = grub-core/kern/list.c;
-  common = grub-core/kern/misc.c;
-  common = grub-core/tests/lib/test.c;
-  common = grub-core/lib/priority_queue.c;
-  ldadd = libgrubmods.a;
-  ldadd = libgrubgcry.a;
-  ldadd = libgrubkern.a;
-  ldadd = grub-core/lib/gnulib/libgnu.a;
-  ldadd = '$(LIBDEVMAPPER) $(LIBZFS) $(LIBNVPAIR) $(LIBGEOM)';
-  condition = COND_HAVE_CXX;
-};
-
-program = {
-  testcase = native;
-  name = cmp_test;
-  common = tests/cmp_unit_test.c;
-  common = tests/lib/unit_test.c;
-  common = grub-core/kern/list.c;
-  common = grub-core/kern/misc.c;
-  common = grub-core/tests/lib/test.c;
-  ldadd = libgrubmods.a;
-  ldadd = libgrubgcry.a;
-  ldadd = libgrubkern.a;
-  ldadd = grub-core/lib/gnulib/libgnu.a;
-  ldadd = '$(LIBDEVMAPPER) $(LIBZFS) $(LIBNVPAIR) $(LIBGEOM)';
-};
-
 program = {
   name = grub-menulst2cfg;
   mansection = 1;
